packages/spectacular/src/lib/feature-test-harness/create-feature-test-harness.ts

```diff
@@ -1,33 +1,27 @@
 import { Location } from '@angular/common';
-import { NgZone, Provider, Type } from '@angular/core';
-import {
-  ComponentFixture,
-  fakeAsync,
-  TestBed,
-  tick,
-} from '@angular/core/testing';
+import { NgZone } from '@angular/core';
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
 import { NavigationExtras, Router, UrlTree } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
 
-import { TestRootComponent, TestRootScam } from './test-root';
-import {
-  createLeftMouseClick,
-  ensureLeadingCharacter,
-  stripLeadingCharacter,
-} from './util';
-
-export interface CreateFeatureTestHarnessOptions {
-  readonly featureModule: Type<unknown>;
-  readonly featurePath: string;
-  readonly providers?: Provider[] | Array<Provider | Provider[]>;
-}
-
+import { TestRootComponent, TestRootScam } from '../test-root';
```

Comment:
Expand barrel imports to void production build issues.

Reply:
✅ Already did.